### DISCUSSION TITLE
Python for Earth Science Students

### DISCUSSION THREAD

**Discussion by kmpaul** [ProjectPythia/projectpythia.github.io#84] [Ideas]
I just found this resource.  We should link it to the Pythia Gallery.

https://github.com/ltauxe/Python-for-Earth-Science-Students

**Comment by brian-rose** [comment]
There's some great content there. At a minimum, we should add to our [external links gallery](https://projectpythia.org/pages/links.html). Why not use it as a guinea pig for the link-adding form that @andersy005 is developing over at #86.

I wonder if we could also adapt some of these lectures for our Foundations book. The material is CC-BY licensed for remixing and reuse. But it would terrific to engage with the author @ltauxe about this!

**Reply by andersy005** [reply]
>  Why not use it as a guinea pig for the link-adding form that @andersy005 is developing over at #86. 

I am about to do so
